### PR TITLE
Journalist can attach image

### PR DESCRIPTION
**Pull request by FacundoOs**
# This PR contains
## [PT](https://www.pivotaltracker.com/story/show/175113661)

- Feature test
- Installing cypress file upload
- Image form and upload button
- Base64 module

### CODE REVIEW

File: src/components/ArticleForm.jsx [hunk 1]
```diff
@@ -2,24 +2,36 @@ import React, { useState } from "react";
...
     const result = await Article.create(
-      e.target.title.value,
-      e.target.teaser.value,
-      e.target.content.value,
+      title,
+      teaser,
+      content,
       selectedCategory,
-      e.target.premium.checked
+      premium,
```
Comment (ctl-lab) [inline src/components/ArticleForm.jsx:34]: Did you check that this take true/false values with the new way of writing =e.target?

**Review by ctl-lab — APPROVED** [review]
Ok, if the new way of writing =e.target works. Which seems to be good now that I saw the module. Thank you @jennymalmefeldt for the answer.

File: src/components/ArticleForm.jsx [hunk 2]
```diff
@@ -2,24 +2,36 @@ import React, { useState } from "react";
...
+    setImage(e.target.files[0]);
+  };
+
   const handleCategoryChange = (value) => {
     setSelectedCategory(value);
   };
 
   const onSubmit = async (e) => {
     e.preventDefault();
+    let { title, teaser, content, premium, encodedImage } = e.target;
```
Comment (emtalen) [inline src/components/ArticleForm.jsx]: move encodedImage out of here, that should be its own `let`.
So just do `let encodedImage` on the line bellow

Comment (jennymalmefeldt) [reply src/components/ArticleForm.jsx]: Moved it out!

**Review by emtalen — APPROVED** [review]

**Pull request merged**